<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Verteste</class>
 <widget class="QMainWindow" name="Verteste">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>626</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Verteste</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" alignment="Qt::AlignLeft">
     <widget class="QFrame" name="projFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="projComboBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="projLabel">
         <property name="text">
          <string>Projeto ativo</string>
         </property>
         <property name="buddy">
          <cstring>projComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="projTimeLabel">
         <property name="text">
          <string>Criado em</string>
         </property>
         <property name="buddy">
          <cstring>projTimeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="projIdLabel">
         <property name="text">
          <string>ID</string>
         </property>
         <property name="buddy">
          <cstring>projIdLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QTreeView" name="projTreeView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="projDescrLabel">
         <property name="text">
          <string>Descrição</string>
         </property>
         <property name="buddy">
          <cstring>projDescrTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="projIdLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="projTimeLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="projListsLabel">
         <property name="text">
          <string>Listas</string>
         </property>
         <property name="buddy">
          <cstring>projTreeView</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTextEdit" name="projDescrTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="listFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="listLabel">
         <property name="text">
          <string>Lista ativa</string>
         </property>
         <property name="buddy">
          <cstring>listComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="listIdLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="listIdLabel">
         <property name="text">
          <string>ID</string>
         </property>
         <property name="buddy">
          <cstring>listIdLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="listTimeLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="listComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="listTimeLabel">
         <property name="text">
          <string>Criado em</string>
         </property>
         <property name="buddy">
          <cstring>listTimeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTableView" name="listTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="listLinesLabel">
         <property name="text">
          <string>Linhas</string>
         </property>
         <property name="buddy">
          <cstring>listTableView</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovoProjeto"/>
    <addaction name="actionAbrirProjeto"/>
    <addaction name="actionAlterarProjeto"/>
    <addaction name="actionSalvarProjeto"/>
    <addaction name="actionFecharProjeto"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionNovaLista"/>
    <addaction name="actionAlterarLista"/>
    <addaction name="actionRemoverLista"/>
    <addaction name="separator"/>
    <addaction name="actionInserirLinha"/>
    <addaction name="actionAlterarLinha"/>
    <addaction name="actionRemoverLinha"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionManual"/>
    <addaction name="actionHistorico"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovoProjeto">
   <property name="text">
    <string>Novo projeto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionAbrirProjeto">
   <property name="text">
    <string>Abrir projeto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalvarProjeto">
   <property name="text">
    <string>Salvar projeto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNovaLista">
   <property name="text">
    <string>Nova lista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionInserirLinha">
   <property name="text">
    <string>Inserir linha</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRemoverLinha">
   <property name="text">
    <string>Remover linha</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRemoverLista">
   <property name="text">
    <string>Remover lista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAlterarLinha">
   <property name="text">
    <string>Alterar linha</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAlterarProjeto">
   <property name="text">
    <string>Alterar projeto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionAlterarLista">
   <property name="text">
    <string>Alterar lista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionHistorico">
   <property name="text">
    <string>Histórico</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionFecharProjeto">
   <property name="text">
    <string>Fechar projeto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
